README.md

```diff
@@ -38,6 +38,8 @@ Local development: set values in .env
 | Name                    | Default | Description                 |
 | ----------------------- | ------- | --------------------------- |
 | CHROMATIC_PROJECT_TOKEN | ``      | Token for Chromatic builds. |
+| RINKEBY_RPC_PATH        | ``      | Alchemy or RPC path.        |
+| MAINNET_RPC_PATH        | ``      | Alchemy or RPC path.        |
```

Comment:
Just realized we should move the RPC vars for each env into .env for hardhat (since it doesn’t have access to the Next.js public env vars) and have another single RPC for the dApp to use for it’s provider.

That also allows a dev to specify their own RPC for solidity/hardhat stuff without deviating from the dApp provider.